packages/dso-toolkit/src/components/tabs/tabs.variables.scss

```diff
@@ -7,10 +7,10 @@ $text-color: colors.$grijs-60;
 $border-radius: 4px;
 $border-color: colors.$grijs-10;
 $link-hover-border-color: colors.$grijs-10;
-$height: 56px;
-$horizontal-padding: units.$u2 - 1px;
-$vertical-padding: units.$u3 - 1px;
-$margin-bottom: units.$vertical-spacing-large;
+$block-size: 56px;
+$inline-padding: units.$u2 - 1px;
```

Comment:
```suggestion
$inline-padding: units.$u3 - 1px;
```

Reply:
Ik ga nu even helemaal los op dit punt van variabelen. 🤣 

Ik stel voor dat de naam van de variabele de CSS-property overneemt of volgt, dus `$padding-inline`.

Reply:
Dat zit me niet zo in de weg. als je daar streng in bent, ga je vroeg of laat krakend vastlopen op het moment dat je meer in die variabele kwijt wil.